BUILDING.md

```diff
@@ -75,22 +86,26 @@ Node.js does not support a platform version if a vendor has expired support
 
-| System       | Support type | Version                         | Architectures    | Notes                         |
-| ------------ | ------------ | ------------------------------- | ---------------- | ----------------------------- |
-| GNU/Linux    | Tier 1       | kernel >= 2.6.32, glibc >= 2.12 | x64, arm         |                               |
-| GNU/Linux    | Tier 1       | kernel >= 3.10, glibc >= 2.17   | arm64            |                               |
-| macOS/OS X   | Tier 1       | >= 10.11                        | x64              |                               |
-| Windows      | Tier 1       | >= Windows 7/2008 R2/2012 R2    | x86, x64         | [1](#fn1),[2](#fn2),[3](#fn3) |
-| SmartOS      | Tier 2       | >= 16                           | x64              |                               |
-| FreeBSD      | Tier 2       | >= 11                           | x64              |                               |
-| GNU/Linux    | Tier 2       | kernel >= 3.13.0, glibc >= 2.19 | ppc64le >=power8 |                               |
-| AIX          | Tier 2       | >= 7.1 TL04                     | ppc64be >=power7 |                               |
-| GNU/Linux    | Tier 2       | kernel >= 3.10, glibc >= 2.17   | s390x            |                               |
-| GNU/Linux    | Experimental | kernel >= 2.6.32, glibc >= 2.12 | x86              | limited CI                    |
-| Linux (musl) | Experimental | musl >= 1.0                     | x64              |                               |
-
-<em id="fn1">1</em>: Tier 1 support for building on Windows is only on 64-bit
-  hosts. Support is experimental for 32-bit hosts.
+| Operating System | Architectures    | Versions                        | Support Type | Notes                             |
+| ---------------- | ---------------- | ------------------------------- | ------------ | --------------------------------- |
+| GNU/Linux        | x64              | kernel >= 3.10, glibc >= 2.17   | Tier 1       | e.g. Ubuntu 16.04, Debian 9, EL 7 |
+| GNU/Linux        | x64              | kernel >= 3.10, musl >= 1.1.19  | Tier 2       | e.g. Alpine 3.8                   |
+| GNU/Linux        | x86              | kernel >= 3.10, glibc >= 2.17   | Experimental | Downgraded as of Node.js 10       |
+| GNU/Linux        | arm64            | kernel >= 4.5, glibc >= 2.17    | Tier 1       | e.g. Ubuntu 16.04, Debian 9, EL 7, [1](#fn1) |
+| GNU/Linux        | armv7            | kernel >= 4.14, glibc >= 2.24   | Tier 1       | e.g. Ubuntu 18.04, Debian 9       |
+| GNU/Linux        | armv6            | kernel >= 4.14, glibc >= 2.24   | Experimental | Downgraded as of Node.js 12       |
+| GNU/Linux        | ppc64le >=power8 | kernel >= 3.13.0, glibc >= 2.19 | Tier 2       | e.g. Ubuntu 16.04, EL 7           |
+| GNU/Linux        | s390x            | kernel >= 3.10.0, glibc >= 2.17 | Tier 2       | e.g. EL 7                         |
```

Comment:
@nodejs/platform-s390 please confirm the kernel and glibc numbers are correct, this is copypasta from the last version since we're on EL 7 still.

Reply:
Yes it should be the same as last release.